nl.jknaapen.fladder.yaml

```
@@ -0,0 +1,234 @@
+id: nl.jknaapen.fladder
+runtime: org.gnome.Platform
+runtime-version: '48'
```

```suggestion
runtime-version: '49'
```

then you can drop ffmpeg-full